docs/api/event-brokers.rst

```diff
@@ -8,9 +8,23 @@ Event Brokers
 
 .. edit-link::
 
-Rasa Core allows you to stream events to a message broker. The event broker
-emits events into the event queue. It becomes part of the ``TrackerStore``
-which you use when starting an ``Agent`` or launch ``rasa.core.run``.
+When running a Rasa Open Source assistant in production, you might want to
+connect your assistant to other services so that they can analyze the data
+that comes from your conversations. For example, you could `connect your
+live assistant to Rasa X <https://rasa.com/docs/rasa-x/installation-and-setup/existing-deployment/>`_
+to review and annotate conversations, or forward messages to an external analytics
+service. 
+
+To do so, Rasa allows you to stream events to an event broker. The event broker
+uses a message streaming service, also known as a message broker,
+to forward Rasa :ref:`events` from the Rasa server to other services.
```

Comment:
I don't think we need to differentiate between "event broker" and "message broker", and probably pick one and use it throughout. they're synonymous IMO

Reply:
See this thread: https://rasa-hq.slack.com/archives/C7FCS16SD/p1583332500132700?thread_ts=1583284919.127600&cid=C7FCS16SD, people don't know what an event broker is -- i'll let @karen-white decide on the phrasing here